### ISSUE TITLE
My Quest is Broken!

### ISSUE THREAD

**Issue by assassinmaster** [Wynncraft/Issues#8859]
I was beginning to do the Mixed Feelings quest but when I've talked to Javen to start the quest, I couldn't talk to him again and I was stuck inside an invisible box. The quest said to talk to Javen again but I couldn't so I've killed myself since I was stuck inside the invisible box. Now I can't do the quest because I can't talk to Javen still. Please help me out here and thank you.

**Comment by Slime1480** [comment]
I think is the same problem as in another issue like this.
As Aroneti said,
Try going to 1.8

**Comment by assassinmaster** [comment]
I'm still stuck!!! but this time, I'm back inside the invisible box again even though I wasn't since I've killed myself. Please, I really need help!

**Comment by ittnoraA** [comment]
Have you tried swapping to a fresh world and trying the section again? If not, I'd recommend giving it a shot.

**Comment by assassinmaster** [comment]
Nope! I have tried and I'm still stuck inside the invisible box unable to do my quest by talking to Javen. Is there a way to restart the quest?
